manifest.yml

```diff
@@ -23,22 +23,16 @@ platforms:
 # - os: darwin
 #   arch: amd64
 terraform_binaries:
-- name: terraform
```

Comment:
Removing these limits the Terraform versions that can be upgraded from. While I'm not opposed to this, I feel that's it warrants a discussion and separate commit message.

Reply:
Already answered in the commit message:
```
cannot be upgraded from versions of terraform < 1.1. This is due to the
usage of a new keyword `moved`. Hence removing old versions of terraform
as that upgrade path is not valid anymore.
```